WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/WebView.swift

```diff
@@ -93,5 +105,23 @@ struct WebView: UIViewRepresentable {
         func webView(_ webView: WKWebView, didCommit navigation: WKNavigation!) {
             parent.onCommit?(webView)
         }
+
+        override func observeValue(forKeyPath keyPath: String?,
+                                   of object: Any?,
+                                   change: [NSKeyValueChangeKey: Any]?,
+                                   context: UnsafeMutableRawPointer?) {
+            guard let object = object as? WKWebView,
+                  object == parent.webView,
+                let keyPath = keyPath else {
+                    return
+            }
+            switch keyPath {
+                case #keyPath(WKWebView.estimatedProgress):
+                    parent.progressView.progress = Float(parent.webView.estimatedProgress)
+                    parent.progressView.isHidden = parent.webView.estimatedProgress == 1
+                default:
+                    assertionFailure("Observed change to web view that we are not handling")
+                }
+        }
```

Comment:
This is redundant, the observing has already been handled in [`WebProgressView`](https://github.com/woocommerce/woocommerce-ios/blob/0c3b9271d8e37edbff2e9ab7992b053a8840e2a5/WooCommerce/Classes/Authentication/WebAuth/WebProgressView.swift#L41-L54).

Reply:
Handled in https://github.com/woocommerce/woocommerce-ios/pull/11501/commits/e7bc63aa1f7045674c688c6fa8652fb13a67c5ad

Reply:
Thanks for this! I was adding this first before adding the `progressView.observeProgress()` and had naively assumed they were linked.